test/parallel/test-whatwg-url-searchparams-sort.js

```diff
@@ -55,12 +55,23 @@ const { test, assert_array_equals } = common.WPT;
 /* eslint-enable */
 
 // Tests below are not from WPT.
-;[
+
+// Test bottom-up iterative stable merge sort
+const tests = [{input: '', output: []}];
+for (let i = 0; 50 > i; i++) {
+  tests[0].input += 'a=b&';
+  tests[0].output.push(['a', 'b']);
```

Comment:
That's right indeed. To use the random list to sort them actually, I've updated it with your comment. Thanks!